distribution/docs/src/main/resources/content/_configuring/content-directory-monitor.adoc

```diff
@@ -0,0 +1,102 @@
+:title: Content Directory Monitor
+:type: configuration
+:status: published
+:parent: Configuring Data Management
+:order: 03
+:summary: Content Directory Monitor.
+
+== {title}
+
+The Content Directory Monitor (CDM) provides the capability to easily add content and metacards into the ${ddf-catalog} by placing a file in a directory.
+
+=== Installing the Content Directory Monitor
+
+The Content Directory Monitor is installed by default with a standard installation of the ${ddf-catalog} application.
+
+=== Configuring the Content Directory Monitor
+
+Configure the CDM from the ${admin-console}:
+
+. Navigate to the *${admin-console}*.
+. Select the *${ddf-catalog}* application.
+. Select the *Configuration* tab.
+. Select *Catalog Content Directory Monitor*.
+
+.Content Directory Monitor Permissions
+[IMPORTANT]
+====
+When setting monitored directory, make sure to set permissions on the new directory to allow ${branding} to access it and to add it to the security manager policy file.
```

Comment:
Should this still mention the security manager?

Reply:
removed